tests/backup/backup_kubevirt_test.go

```diff
@@ -1760,3 +1761,213 @@ var _ = Describe("{KubevirtInPlaceRestoreWithReplaceAndRetain}", func() {
+			log.InfoD("Deleting the kubevirt VMs from the namespace")
+			for _, namespace := range bkpNamespaces {
+				err := DeleteAllVMsInNamespace(namespace)
+				dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying deletion of kubevirt VMs from the namespace [%s]", namespace))
+			}
+		})
+
+		Step("Verify next non namespace labelled scheduled backup is failed state after VM deletion", func() {
+			log.InfoD("Verify next non namespace labelled scheduled backup is failed state after VM deletion")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			for _, namespace := range bkpNamespaces {
+				schBackupAfterVMDeletion, err := GetNextScheduleBackupName(nonLabelSchNamespaceMap[namespace], time.Duration(periodicSchedulePolicyInterval), ctx)
+				dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying fetching next schedule backup [%s] after VM deletion ", schBackupAfterVMDeletion))
+				bkpStatus, bkpReason, err := Inst().Backup.GetBackupStatusWithReason(schBackupAfterVMDeletion, ctx, BackupOrgID)
+				dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying backup [%s] status  ", schBackupAfterVMDeletion))
+				if bkpStatus == api.BackupInfo_StatusInfo_Failed { //TODO add the string check
+					log.Infof(fmt.Sprintf("The backup failed when the schedule VMs were deleted from namespace, Reason [%s]", bkpReason))
+				} else {
+					err := fmt.Errorf("The backup [%s] is still in success state when expected to fail , Reason [%s]", schBackupAfterVMDeletion, bkpReason)
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying the status of scheduled backup [%s]", schBackupAfterVMDeletion))
+				}
+			}
+		})
+
+		Step("Verify next namespace labelled scheduled backup is in success state after VM deletion", func() {
+			log.InfoD("Verify next namespace labelled scheduled backup is in success state after VM deletion")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			for _, namespace := range bkpNamespaces {
+				schBackupAfterVMDeletion, err := GetNextScheduleBackupName(labelSchNamespaceMap[namespace], time.Duration(periodicSchedulePolicyInterval), ctx)
+				dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying fetching next schedule [%s] backup after VM deletion ", schBackupAfterVMDeletion))
+				bkpStatus, bkpReason, err := Inst().Backup.GetBackupStatusWithReason(schBackupAfterVMDeletion, ctx, BackupOrgID)
+				dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying backup [%s] status  ", schBackupAfterVMDeletion))
+				if bkpStatus == api.BackupInfo_StatusInfo_Success {
+					log.Infof("The backup is success state for labelled scheduled backup [%s] after VM delete")
+				} else {
+					err := fmt.Errorf("The backup failed for labelled scheduled backup [%s] after VM delete : Reason [%s]", schBackupAfterVMDeletion, bkpReason)
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying the status of scheduled backup [%s]", schBackupAfterVMDeletion))
+				}
+			}
```

Comment:
Create VMs again and check that it is getting picked in the next scheduled backup

Reply:
Added step to recreate VM and verify both labelled and non labelled schedule backup.

https://jenkins.pwx.dev.purestorage.com/job/portworx-backup/job/system-tests/job/byoc/job/px-backup-on-demand-system-test-byoc/4902/consoleFull
https://aetos.pwx.purestorage.com/resultSet/testSetID/585239/testCaseID/3560058